pkg/api/converterfromadmin.go

```diff
@@ -300,7 +300,12 @@ func convertIdentityProviderAdmin(in admin.IdentityProvider, old *IdentityProvid
 
 func mergeConfig(oc *admin.OpenShiftManagedCluster, cs *OpenShiftManagedCluster) {
 	in, out := oc.Config, &cs.Config
-
+	if in.ClusterVersion != nil {
```

Comment:
Why this one was added? I think we dont propagate this field back to OpenShiftManageCluster?

Reply:
Or my brains still on holiday?

Reply:
this PR added a round trip test to prove that you can convert admin -> internal -> admin and get the same thing back - this discovered several missing fields which are added here